controller/templates/confd_settings.py

```diff
@@ -46,3 +46,14 @@
 {{ end }}
 
 UNIT_HOSTNAME = '{{ or (.deis_controller_unitHostname) "default" }}'
+
+# AUTH
+# LDAP
+LDAP_ENDPOINT = '{{ .deis_controller_auth_ldap_endpoint }}'
```

Comment:
You'll have to make these conditional or else confd will bail forever when these keys are unset: https://ci.deis.io/job/test-ec2/213/